install/template.html

```diff
@@ -98,6 +98,7 @@
 <h4><?php echo ts('CiviCRM Database Settings'); ?></h4>
 <p style="margin-left: 2em" id="mysql_credentials">
   <label for="mysql_server"> <span><?php echo ts('MySQL server:'); ?></span> <input id="mysql_server" type="text" name="mysql[server]" value="<?php echo $databaseConfig['server']; ?>" /></label> <br />
+  <span><?php echo ts('If your mysql server is running on other port then default 3306, provide server info as server:port (i.e. localhost:1234) ') ?> </span> </br>
```

Comment:
I think the message should be `(If your mysql server is running on a port other than default 3306, provide server info as a server:port (i.e. localhost:1234))`.
Note the typo then**.